runtime/laos/src/configs/transaction_payment.rs

```diff
@@ -16,12 +16,20 @@ impl WeightToFeePolynomial for LengthToFee {
 	type Balance = Balance;
 
 	fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
-		smallvec![WeightToFeeCoefficient {
-			degree: 1,
-			coeff_frac: Perbill::zero(),
-			coeff_integer: TRANSACTION_BYTE_FEE,
-			negative: false,
-		},]
+		smallvec![
+			WeightToFeeCoefficient {
+				degree: 1,
+				coeff_frac: Perbill::zero(),
+				coeff_integer: TRANSACTION_BYTE_FEE,
+				negative: false,
+			},
+			WeightToFeeCoefficient {
+				degree: 3,
+				coeff_frac: Perbill::zero(),
+				coeff_integer: 100,
```

Comment:
this was `SUPPLY_FACTOR` before, which is better than just 100 (unclear). So I would suggest to just restore `SUPPLY_FACTOR`

Reply:
what's the meaning of supply factor ?

Reply:
what is this value then? 100? how is it selected, why not 1_000, etc.? Just suggesting `SUPPLY_FACTOR` because at a minimum we have the justification that it is used by some other live chains. We have the same value as `SUPPLY_FACTOR` but we are not just naming it

Reply:
it's the factor used by Moonbeam having a supply of ~ 1B .. like we do.
Moonriver has a supply of ~10M and the SUPPLY_FACTOR is 1.

So the SUPPLY_FACTOR has sense in that context. Purestake use it to adapt to the different supply of their chains. Dunno what's the meaning of that coeff in out case, a part for being a coeff.